### PR TITLE
Add Audit Log chapter (Closes #318, Closes #310)

### LINKED ISSUE #310: Write Security Chapter for Admin Guide
- [x] [Security concepts](https://github.com/SUSE/doc-caasp/issues/311)
- [x] [Managing Users and Groups](https://github.com/SUSE/doc-caasp/issues/312)
- [x] [Role Management (RBAC)](https://github.com/SUSE/doc-caasp/issues/313)
- [x] [LDAP configuration and usage](https://github.com/SUSE/doc-caasp/issues/314)
- [x] [Certificates](https://github.com/SUSE/doc-caasp/issues/315)
- [x] [Pod Security Policies](https://github.com/SUSE/doc-caasp/issues/316)
- [x] [Networking Policies (Cilium)](https://github.com/SUSE/doc-caasp/issues/317)
- [ ] [Security Audit Log](https://github.com/SUSE/doc-caasp/issues/318)

### LINKED ISSUE #318: Describe Security Audit Log
Kubernetes provides a security audit log. Describe how it can be configured, what it is used for, how it can be accessed and how the contents can be interpreted.

### PR DESCRIPTION
**Pull request by c3y1huang**
# Make sure it's tested
Verified:
* k8s v1.17.4
* k8s v1.18.0
* enabled on bootstrap
* enabled on running cluster
* audit-log-maxbackup files are not over created
* audit-log-maxsize not over the configured size
* audit disable stopped audit logging

K8s version: 1.18.0
```
010084073210:~ # ls /var/log/kube-apiserver/
audit-2020-04-28T07-50-51.458.log  audit-2020-04-28T07-57-12.519.log
audit-2020-04-28T07-52-56.119.log  audit-2020-04-28T07-59-10.089.log
audit-2020-04-28T07-55-07.112.log  audit.log
```
K8s version: 1.17.4
```
010084072112:~ # ls /var/log/kube-apiserver/
audit-2020-04-28T07-07-59.194.log  audit-2020-04-28T07-18-15.636.log
audit-2020-04-28T07-10-56.555.log  audit-2020-04-28T07-22-35.683.log
audit-2020-04-28T07-13-56.494.log  audit.log
```

# Describe your changes
This chapter is about Kuberentes Auditing and the procedures to enable/disable the auditing.

# Related Issues / Projects
Please provide links to Bugzilla and other GitHub projects with your description if they are related to the changes.

Relates to:
* https://bugzilla.suse.com/show_bug.cgi?id=1168491
* https://github.com/SUSE/avant-garde/issues/1520
* https://github.com/SUSE/doc-caasp/issues/318

# Enable maintainer updates
Please enable maintainer updates so we can push commits into your branch to make collaboration and reviews easier.

# Do not force push your branch
Please avoid force pushing to branches that are subject of pull requests. Force pushing breaks maintainer commits in many cases and is very hard (if not impossible) to untangle for backporting.

# Labels
Please set any (and all) appropriate labels that describe the status of the PR.

| **Label** | **Description** |
| --- | --- |
| P1 | PR should be worked on and merged as soon as possible |
| Blocked | Work can not proceed because other work has not been completed, PR can not be merged (code has not been merged but documentation is ready) |
| On-Hold | Underlying work is completed but the PR should not be merged |
| ReleaseNotes | User interaction is required after the introduction of this change and the change must be mentioned in the release notes |
| v3/v4/v4.x | Which version of the release the PR should be merged into, this can be multiple versions, please set the "Backport" label if it needs to go into a previous release |
| Needs Review | Some details of the PR are known to be incomplete and must be discussed with other engineers before merging (if possible assign reviewers or cc mention in comments), PR can not be merged |

Signed-off-by: Chin-Ya Huang <chin-ya.huang@suse.com>

### CODE REVIEW

File: adoc/admin-audit.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      path: /etc/kubernetes/policies/audit.yaml
+      type: File
+    name: audit-policy
+  - hostPath:
+      path: /var/log/kube-apiserver
+      type: DirectoryOrCreate
+    name: audit-logs
+====
+
+. Restart kubelet on all master node.
```
[the remark this answers is not in the document]
Comment (c3y1huang) [reply adoc/admin-audit.adoc]: Reworded. Already mentioned the steps need to repeat for every master node. Shouldn't need to mention again. PTAL.

**Comment by c3y1huang** [comment]
@r0ckarong @nkoranova please help with this doc review and merge. Thanks.

**Comment by c3y1huang** [comment]
> I did not see much different steps in `Enable Auditing During Cluster Bootrap` and `Enable Auditing On Running Cluster`. The rests LGTM.

Yes the steps are very similar. I'm leaving that for @r0ckarong and @nkoranova to work on readability and layouts.

**Comment by r0ckarong** [comment]
@nkoranova I did a review of this and added some commits. Please have another look and merge when ready. Thanks!

**Comment by r0ckarong** [comment]
@jordimassaguerpla I guess we can merge this already if it's done? No need to wait for QA approval since this is a k8s feature.

**Pull request merged**